extensions/azurecore/src/extension.ts

```diff
@@ -135,6 +137,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 				new AzureResourceDatabaseProvider(new AzureResourceDatabaseService(), extensionContext),
 				new SqlInstanceProvider(new SqlInstanceResourceService(), extensionContext),
 				new PostgresServerProvider(new PostgresServerService(), extensionContext),
+				// new CosmosDbMongoProvider(new CosmosDbMongoService(), extensionContext)
```

Comment:
Just delete these until you actually enable it.

Although if this code isn't enabled then I'm curious what the point of getting this in now is. Could you not just wait until the feature is ready to do the PR?